gradle.properties

```diff
@@ -69,3 +69,5 @@ sonarVersion=3.3
 repoUrl=https://repo.hortonworks.com/content/repositories/releases/
 springRepoUrl=https://repo.spring.io/libs-release
 cdpRepoUrl=https://repository.cloudera.com/artifactory/cloudera-repos/
+protobufVersion=3.19.1
```

Comment:
nit: version constants are above the repo urls

Reply:
Oh, thanks, updated it.  I have not realised that this file has any structure. I have added some comments to make it explicit.